.eslintrc

```diff
@@ -0,0 +1,140 @@
+env:
+    browser: true
+    node: true
+    es6: true
+    jest: true
+
+globals:
+    $: true
+    widgets: true
```

Comment:
I don't think `widgets` is needed here.  I think that's left over from `nui-widgets`.